### PR TITLE
Gave The Ops Powers Back (2)

### PR DESCRIPTION
**Pull request by Supertre03**

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -765,6 +765,56 @@ groups:
       suffix: '&4'
     inheritance:
     - mortal
+    worlds:
+      survival:
+        permissions:
+        - powers.fireballpunch
+        - powers.wallrun
```
Comment (Supertre03) [inline PermissionsEx/permissions.yml]: I CANT EDIT THIS
HELP ME PLEASE

Comment (Supertre03) [inline PermissionsEx/permissions.yml]: D:

Comment (malonnnn) [inline PermissionsEx/permissions.yml]: i cant help you, this was not made under my account, sorry :\

You should have done each hero separately. next time, do a PR for each hero, it makes it easier when errors happen.

**Comment by malonnnn** [comment]
invalid since we can't edit it

**Pull request closed without merging**